src/applications/vaos/tests/appointment-list/components/ConfirmedAppointmentDetailsPage.unit.spec.jsx

```diff
@@ -325,5 +335,9 @@ describe('VAOS <ConfirmedAppointmentDetailsPage>', () => {
         name: 'We’re sorry. We’ve run into a problem',
       }),
     ).to.be.ok;
+
+    waitFor(() => {
```

Comment:
This needs to have `await` in front

Reply:
good to know `await waitFor` goes together